### PR TITLE
Update changelog for sprint 48

### PR DESCRIPTION
**Pull request by bgfoshay**
## What’s changing?

Update changelog for sprint 48

## Why?

Update changelog for sprint 48

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project **does not** adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.1.48] - 2022-09-06
+
+### Changed
+- Added exceptions to allow "-" and "_" for Participant and Case Ids
```
Comment (dborgen2) [inline CHANGELOG.md:9]: Nitpick (non-blocking): Also limiting the characters to 20 characters from the Query Tool/Find Matches API.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+- Email Notification Azure Function
+- New Table component to the common components library
```
Comment (dborgen2) [inline CHANGELOG.md:13]: Nitpick (non-blocking): Also added Pagination and Numeric input to the common library

**Review by ryangeorgeharvey — APPROVED** [review]
LGTM

**Review by dborgen2 — APPROVED** [review]
LGTM

**Pull request merged**